modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/internal/search/spi/model/permission/test/UserSearchPermissionFilterContributorTest.java

```diff
@@ -93,6 +93,46 @@ public void testWhenHasOrganizationManageAccountsPermissionSearch()
 			_performUserSearchCount(accountEntry.getAccountEntryId(), userA));
 	}
 
+	@Test
+	public void testWhenHasOrganizationManageSuborganizationPermissionSearch() {
+		Organization organization = OrganizationTestUtil.addOrganization();
+
+		Organization suborganization = OrganizationTestUtil.addOrganization(
```

Comment:
Can you pass the `RandomTestUtil.randomString(), false` parameters instead of create a new method in the Util class?